.github/workflows/docker.yml

```diff
@@ -13,8 +13,6 @@ jobs:
           git -C deps/abcBridge submodule update --init
       - run: .github/ci.sh set_version
         id: outputs
-      - uses: satackey/action-docker-layer-caching@v0.0.8
```

Comment:
I've had to remove this in other actions, so I went ahead and removed it in this one. Eventually we probably will want to look around and see if there's a better way to do docker + pushing to github somehow and really diving into that. But for now, this'll work.

Reply:
I've been thinking that caching docker builds would be a great use case for GitHub container registry - especially given that:

> All data transferred out, when triggered by GitHub Actions, and data transferred in from any source is free.

Plus they are probably going to be colocated way better than e.g. caching to dockerhub.

Reply:
That's what I was thinking :)

It'll be a good rabbit hole for me to look into next time I nerd snipe myself